test/fixtures/adopter_applications.yml

```diff
@@ -1,7 +1,7 @@
 # Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
 adopter_application_one:
-  dog: dog_one
+  dog: pending_adoption_one
```

Comment:
I wrote up a long stream of thought on this, but after re-thinking, this is good, too.